Models/User.js

```diff
@@ -49,6 +49,10 @@ const userSchema = new mongoose.Schema({
         type: Date,
         default: +new Date() + 30 * 24 * 60 * 60 * 1000,
     },
+    current_data_registration: {
```

Comment:
date_registration i current nie wiadomo w ogole o co tutaj chodzi 
zamiast current to date_registration
a tamte to np expiration_date
analogicznie pewnie z racji zmiany bedzie do poprawy w walidacji endpointu